tfhe/src/integer/server_key/radix_parallel/tests_unsigned/modulus_switch_compression.rs

```diff
@@ -0,0 +1,42 @@
+use crate::integer::gen_keys_radix;
+use crate::integer::tests::create_parametrized_test;
+#[cfg(tarpaulin)]
+use crate::shortint::parameters::coverage_parameters::*;
+use crate::shortint::parameters::*;
+use rand::Rng;
+
+const NB_TESTS: usize = 10;
+
+create_parametrized_test!(modulus_switch_compression);
+
+fn modulus_switch_compression<P>(param: P)
+where
+    P: Into<PBSParameters>,
+{
+    let param = param.into();
+
+    // Modulus switch compression and multi bit PBS are currently not compatible
+    if let PBSParameters::MultiBitPBS(_) = &param {
+        return;
+    }
```

Comment:
I don't like this it would look like the test was run,

prefer using the manual choice of parameters via

```rust
create_parameterized_test!({
    PARAM_SET1,
    PARAM_SET2,
});
```

then we'll revert the change in the macro once we have the fix for the multibit